### ISSUE TITLE
pygsheets removes values from returned spreadsheet

### ISSUE THREAD

**Issue by richardARPANET** [nithinmurali/pygsheets#428]
For this sheet

https://docs.google.com/spreadsheets/d/1ZqFi0HTFFshwaSGaNL4lkAMT_FJ3izQSBXxaqXCzYGY/edit#gid=0

When doing `sheet.get_as_df(numerize=True)`

This line of code in pygsheets worksheet.py

```
values = [numericise_all(row[:len(values)], empty_value) for row in values]
```

Removes some values for some reason.

Input to that function is

```
[['Location', 'Min Temp', 'Max Temp', 'Sunrise', 'Sunset', 'Image'],
 ['London',
  '19.44',
  '20.56',
  '3:56:30 AM',
  '7:58:36 PM',
  'https://i.imgur.com/MWkipGC.png'],
 ['Paris',
  '17',
  '20.56',
  '3:58:14 AM',
  '7:37:02 PM',
  'https://i.imgur.com/E0B8jfs.jpg']]
```

Output is

```
[['Location', 'Min Temp', 'Max Temp'],
 ['London', 19.44, 20.56],
 ['Paris', 17, 20.56]]
```

As you can see, columns are removed.

**Comment by fstinner** [comment]
see solution here:  https://github.com/nithinmurali/pygsheets/issues/434

**Comment by nithinmurali** [comment]
fixed in #436
